staging/src/k8s.io/cli-runtime/pkg/resource/builder_test.go

```diff
@@ -49,14 +49,13 @@ import (
 	utiltesting "k8s.io/client-go/util/testing"
 
 	// TODO we need to remove this linkage and create our own scheme
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This also changed unrelated line.
Please keep the PR scope simple and clean for the PR message and the corresponding issue.

Reply:
removed